libs/nestjs-admin/src/login.exception.ts

```diff
@@ -0,0 +1,9 @@
+import { HttpException, HttpStatus } from '@nestjs/common'
+
+class LoginException extends HttpException {
+  constructor() {
+    super('Forbidden', HttpStatus.FORBIDDEN)
```

Comment:
Actually we want UNAUTHORIZED, not FORBIDDEN (it's counterintuitive but "unauthorized" means you're missing authentication, forbidden means you're authenticated but not authorized to do this action).